packages/merge-styles/src/IStyle.ts

```diff
@@ -7,6 +7,8 @@ import { IRawStyleBase } from './IRawStyleBase';
  * {@docCategory IRawStyle}
  */
 export interface IRawStyle extends IRawStyleBase {
+  [key: string]: string | number | undefined | Record<string, IStyle>;
```

Comment:
add this in `IRawStyleBase`?

Reply:
side note, would be nice to have this one day: https://github.com/Microsoft/TypeScript/issues/6579

Reply:
i wish

Reply:
hmm. I did this first, but then it put the knowledge of the selectors in `IRawStyleBase`. I *think* this is more correct.